README.md

```diff
@@ -1 +1,170 @@
-# Catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h2> My Catalogs App </h2>
+</div>
+
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 ruby-group-capstone-project ](#-catalog-of-my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of My Things <a name="ruby-group-capstone-project"></a>
+In this project, we created a console app that will help you to keep a record of different types of things you won: books, music albums, movies, and games. Everything based on the UML class diagram. The data stored in JSON files but we also prepared a database with tables structure analogical to our program's classes structure.
+
+- books
+- music albums
+- movies
+- games
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+        <li><a href="https://www.postgresql.org/">PostgreSQL.</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add new book.
+- Add new games.
+- Add new music albums.
+- List all items added.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Video Presentation <a name="#video"></a>
+- [Catalog of my things video Link ](https://drive.google.com/file/d/1NNdMwNiy-Gk8TbgYOkzxiIyv7bDSu5sp/view?usp=sharing)
```

Comment:
- [ ] Nice work so far. Everything is fallen into place beautifully. However, according to the previous requirements, the maximum length of the video should be 5 minutes but yours is 7 minutes 8 seconds. Kindly reduce it to fall within the specification. :pray: 
- [ ] I like the idea of saving the game details into the file. :1st_place_medal: There are a lot of other things you did in this project that you are very proud of. Right? kindly highlight one or more of those in the video. (Remember, it is part of the project requirements). :pray: